Doc/c-api/long.rst

```diff
@@ -13,6 +13,11 @@ All integers are implemented as "long" integer objects of arbitrary size.
 On error, most ``PyLong_As*`` APIs return ``(return type)-1`` which cannot be
 distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
 
+.. seealso:: Python methods :meth:`int.to_bytes` and :meth:`int.from_bytes`
+   (called via one of the :c:func:`PyObject_CallMethod` APIs),
+   to convert a :c:type:`PyLongObject` of arbitrary size
+   to or from an array of bytes.
```

Comment:
```suggestion
   to/from an array of bytes.
```

One minor tweak to my previous suggestion—usually the recommendation is to avoid `/` and use explicit prose instead, but in this case it simplifies parsing this long and fairly complex sentence, IMO.